teachers_digital_platform/js/scroll.test.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const scroll = require( './scroll.js' );
+
+const HTML_SNIPPET = `
+  <a class="one" href="#one" data-scroll>one</a>
+  <a class="two" href="#two" data-scroll>two</a>
+  <a class="three" href="#three" data-scroll>three</a>
+  <div id="one"></div>
+  <div id="two"></div>
+  <div id="three"></div>
+`;
+
+// Helper function to simulate events
+function simulateEvent( eventType, target, eventOption ) {
+  const event = document.createEvent( 'Event' );
+  if ( eventOption && eventOption.keyCode ) {
+    event.keyCode = eventOption.keyCode;
+  }
+  event.initEvent( eventType, true, true );
+  return target.dispatchEvent( event );
+}
+
+/* eslint-disable max-lines-per-function, no-undefined */
+describe( 'Scroll', () => {
+
+  beforeEach( () => {
+    // Load HTML fixture
+    document.body.innerHTML = HTML_SNIPPET;
+    global.scroll = jest.fn();
```

Comment:
`global` = `window`, see https://stackoverflow.com/a/41888736

Reply:
And `window.scroll` is mocked here because `smoothscroll-polyfill` uses it and throws an error if it doesn't exist.